cities/solingen.json

```diff
@@ -0,0 +1,77 @@
+{
+  "crs": {
+    "properties": {
+      "name": "urn:ogc:def:crs:OGC:1.3:CRS84"
+    },
+    "type": "name"
+  },
+  "type": "FeatureCollection",
+  "features": [
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          "7.082048",
+          "51.170615"
+        ]
+      },
+      "properties": {
+        "title": "Solingen-Mitte",
+        "location": "Neumarkt",
+        "opening_hours": "Tu,Th,Sa 07:00-14:00"
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          "6.998576",
+          "51.162735"
+        ]
+      },
+      "properties": {
+        "title": "Ohligs",
+        "location": "Ohligser Markt",
+        "opening_hours": "Tu,Sa 07:00-14:00"
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          "7.040854",
+          "51.185362"
+        ]
+      },
+      "properties": {
+        "title": "Wald",
+        "location": "Walder Marktplatz",
+        "opening_hours": "We,Fr 07:00-14:00"
+      }
+    },
+    {
+      "type": "Feature",
+      "geometry": {
+        "type": "Point",
+        "coordinates": [
+          "7.062223",
+          "51.151896"
+        ]
+      },
+      "properties": {
+        "title": "Höhscheid",
+        "location": "Peter-Höfer-Platz",
+        "opening_hours": "Th 07:00-14:00"
+      }
+    }
+  ],
+  "metadata": {
+    "data_source": {
+      "title": "Stadt Solingen",
+      "url": "http://www2.solingen.de/C12572F80037DB19/html/ACDC40511C247E79C1257473002CF1B1?opendocument&nid1=14336_71684"
```

Comment:
# Branch update
- I merged your commits into a single commit.
- I fixed the copy/paste error in the _README.md_.
- I rebased your branch on top of our latest commit on `master`.
- I removed the `map_initialization` block in _solingen.json_ since it is no longer needed.
- I force pushed your branch which automatically updated this pull request.

# Remaining issues
- The URL in the _README.md_ and in _solingen.json_ needs to be updated.
- Since the pull request is somewhat old please check if the markets are still up-to-date.